rig-neo4j/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "rig-neo4j"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+readme = "README.md"
+description = "Neo4j implementation of a Rig vector store."
+repository = "https://github.com/0xPlaygrounds/rig"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = "0.3.30"
+neo4rs = "0.8.0"
+rig-core = { path = "../rig-core", version = "0.3.0" }
+serde = { version = "1.0.203", features = ["derive"] }
+serde_json = "1.0.117"
+tracing = "0.1.40"
+textwrap = { version = "0.16.1", optional = true }
+term_size = { version = "0.3.2", optional = true }
```

Comment:
Move these dependencies to the `dev-dependencies` since they are not used in the actual crate. Note: `dev-dependencies` apply to both tests and examples!